src/main/java/org/apache/xmlbeans/XmlError.java

```diff
@@ -51,6 +51,7 @@
     private final int _column;
     private int _offset = -1;
 
+    // TODO Never disposed?
```

Comment:
JIRA is used for tracking issues - if you don't intend to do anything about this - can you log a JIRA and replace the TODO comment with a comment that provides a link to the JIRA?